shell/platform/android/test/io/flutter/embedding/android/FlutterActivityAndFragmentDelegateTest.java

```diff
@@ -362,7 +394,7 @@ public void itExecutesDartLibraryUriProvidedByHost() {
     delegate.onStart();
 
     verify(mockFlutterEngine.getDartExecutor(), times(1))
-        .executeDartEntrypoint(eq(expectedEntrypoint));
+        .executeDartEntrypoint(eq(expectedEntrypoint), any(List.class));
```

Comment:
Shouldn't we be calling executeDartEntrypoint with null for the second argument?  Please switch to a null matcher.

Reply:
done